sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs

```diff
@@ -1576,13 +1701,27 @@ public partial class ResourceDeleteSuccessEventData
         public string SubscriptionId { get { throw null; } }
         public string TenantId { get { throw null; } }
     }
+    public partial class ResourceHttpRequest
+    {
+        internal ResourceHttpRequest() { }
+        public string ClientIpAddress { get { throw null; } }
+        public string ClientRequestId { get { throw null; } }
+        public string Method { get { throw null; } }
+        public string Url { get { throw null; } }
```

Comment:
This could be a Uri but we already have several other system events that use a string `url`